app/controllers/api/v1/exchange_rates_controller.rb

```diff
@@ -0,0 +1,79 @@
+module Api
+  module V1
+    ##
+    # @api {get} /api/v1/exchange_rates Obtener histórico de tipos de cambio
+    # @apiName GetExchangeRates
+    # @apiGroup ExchangeRates
+    #
+    # @apiParam {String} from_currency Código de moneda origen (obligatorio)
+    # @apiParam {String} to_currency Código de moneda destino (obligatorio)
+    # @apiParam {Date} [start_date] Fecha de inicio (formato YYYY-MM-DD, predeterminado: hace 30 días)
+    # @apiParam {Date} [end_date] Fecha de fin (formato YYYY-MM-DD, predeterminado: fecha actual)
+    # @apiParam {Integer} [page] Número de página (predeterminado: 1)
+    # @apiParam {Integer} [per_page] Elementos por página (predeterminado: 30, máximo: 100)
+    #
+    # @apiSuccess {Object} meta Metadatos de la consulta y paginación
+    # @apiSuccess {Object[]} data Lista de tipos de cambio históricos
+    #
+    # @apiError {Object} error Mensaje de error si faltan parámetros obligatorios
+    #
+    # @apiExample Example:
+    #   GET /api/v1/exchange_rates?from_currency=USD&to_currency=EUR&start_date=2023-01-01
+    class ExchangeRatesController < ApiController
+      def index
+        # Inicializar parámetros con valores por defecto
+        from_currency = params[:from_currency]
+        to_currency = params[:to_currency]
+        start_date = params[:start_date].present? ? Date.parse(params[:start_date]) : 30.days.ago.to_date
+        end_date = params[:end_date].present? ? Date.parse(params[:end_date]) : Date.current
+        
+        # Validar parámetros requeridos
+        unless from_currency.present? && to_currency.present?
```

Comment:
When possible, avoid negative conditions. Positive conditions are always easier to read:
```suggestion
        if from_currency.blank? || to_currency.blank?
```